backend/app/infrastructure/routes/allocations.py

```diff
@@ -241,12 +242,20 @@ def post(self, user_address: str):
     },
 )
 class EpochAllocations(OctantResource):
+    @ns.param(
+        "includeZeroAllocations",
+        description="Include zero allocations to projects. Defaults to false.",
+        _in="query",
+    )
     @ns.marshal_with(epoch_allocations_model)
     @ns.response(200, "Epoch allocations successfully retrieved")
     def get(self, epoch: int):
+        include_zero_allocations = request.args.get("includeZeroAllocations", type=bool)
```

Comment:
As far as i'm correct, I think that ```request.args.get("includeZeroAllocations", type=bool)``` returns None, not False when argument isn't specified. So we'll be passing include_zero_allocations as None to get_all_allocations. We need additional mapping for that, i guess.